salesagility/SuiteCRM
# Fixed error message css + email warning config option

## Description



Fixed the styling of the error message:

![error](https://user-images.githubusercontent.com/26431166/61615916-14a8fb00-ac5f-11e9-8521-d7aa76ea08fb.png)

![fixed](https://user-images.githubusercontent.com/26431166/61615926-183c8200-ac5f-11e9-9b12-20b125fbd65c.png)


## How To Test This

1. Install the CRM without setting up SMTP settings
2. View the error message in all 4 themes.
3. Check that the new checkbox field email warning notification is visible in email settings.
4. Check that the user can disable email warnings.

## Types of changes


### Final checklist
